doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multi-Orchestrator Example
+==========================
+SmartSim offers functionality to automate the deployment of multiple
+databases on an HPC cluster, supporting workloads that require multiple
+``Orchestrators`` for a ``Experiment``. For instance, a workload may consist of a
+simulation with high inference performance demands (necessitating a co-located deployment),
+along with an analysis and
+visualization workflow connected to the simulation (requiring a standard orchestrator).
+
+Below is a simple example demonstrating the process of:
+
+1. Launch two standard Orchestrators within the experiment with unique identifiers
+2. Launch one colocated Orchestrator within the experiment with a unique identifier
+3. Connect SmartRedis clients within the driver script to send tensors to std Orchestrators
+4. Connect SmartRedis clients within the application to retrieve tensors from std databases to store in colo databases
+
+The Application Script
+----------------------
+To store and retrieve data to and from databases
+during the course of an application, you must
+initialize multiple SmartSim clients (SmartRedis).
+In this section, we write an application script
+to demonstrate how to connect SmartRedis
+clients in the context of multiple
+launched databases. Using the clients, we retrieve tensors
+from two databases launched in the driver script, then store
+the tensors in the colocated database.
+
+To begin, import the necessary packages:
+
+.. code-block:: python
+
+  from smartredis import ConfigOptions, Client, log_data
+
+The SmartRedis ``log_data`` will be used to monitor the application status
+through the course of the Experiment.
+
+Initialize the Clients
+^^^^^^^^^^^^^^^^^^^^^^
+To establish a connection with each database,
+we need to initialize a new SmartRedis client for each
+``Orchestrator``.
+
+Step 1: Initialize ConfigOptions
+""""""""""""""""""""""""""""""""
+Since we are launching multiple databases within the experiment,
+the SmartRedis ``ConfigOptions`` object is required when initializing
+a client in the application.
+We use the ``ConfigOptions.create_from_environment()``
+function to create three instances of ``ConfigOptions``,
+with one instance associated with each launched ``Orchestrator``.
+Most importantly, to associate each launched Orch to a ConfigOptions object,
+the ``create_from_environment()`` function requires specifying the unique database identifier
+argument named `db_identifier`. The `db_identifier` argument
+serves as a unique identifier for each database launched, guiding
```

Comment:
You've described db_identifier as the `unique database identifier` twice in the last two sentences. Recommend deleting one or the other

Reply:
thank you for catching, trying to eliminate all fluff